template-compose/buildSrc/src/main/java/Versions.kt

```diff
@@ -42,10 +41,8 @@ object Versions {
     const val DETEKT_VERSION = "1.21.0"
 
     // Testing libraries
-    const val TEST_ANDROIDX_CORE_VERSION = "1.4.0"
-    const val TEST_JUNIT_ANDROIDX_EXT_VERSION = "1.1.2"
     const val TEST_JUNIT_VERSION = "4.13.2"
     const val TEST_KOTEST_VERSION = "4.6.3"
-    const val TEST_MOCKK_VERSION = "1.10.6"
-    const val TEST_RUNNER_VERSION = "1.4.0"
+    const val TEST_MOCKK_VERSION = "1.13.3"
```

Comment:
Any reason why these libraries' versions are not the same between the template and sample? 🤔

Reply:
@chornerman As explained in the PR's description, we need to use MockK `1.12.3` to be able to mock in the instrument testing for the `sample`. With the `template` project, we will run UI tests with Robolectric so no MockK mocking in the instrument test because Robolectric tests are actually unit tests.

For summary:
- We have to use `MockK 1.12.3` for the sample.
- We can use the latest `MockK for the template, even `1.13.4` 👉 updating

Reply:
@luongvo Got it now. Thanks for the clarification and sorry for missing the PR's description 🙏🏼